PowerShell Scanners/LLMNR Enabled/LLMNR Enabled.ps1

```diff
@@ -0,0 +1,14 @@
+
+$Multicast = Get-ItemProperty -Path 'HKLM:SOFTWARE\Policies\Microsoft\Windows NT\DNSClient\' -Name 'EnableMulticast'
+
+
+if( $Multicast.EnableMultiCast -eq 0){
+  $Enabled = $false
+} else{
+    $Enabled = $true
+}
```

Comment:
One last issue, if the registry key doesn't exist it will fail this check and return data as if it's enabled.

I'm not sure if that is the case. if a computer doesn't have this key set at all (like mine for example, it doesn't exist unless it's created) does that mean LLMNR is enabled or disabled?

If it's disabled when the key is missing or set to 0, please use this code instead.

```
if( $Multicast.EnableMultiCast -eq 1){
    $Enabled = $true
} else{
    $Enabled = $false
}
```

Otherwise if it's enabled even when the registry key isn't set, this can be merged.

Reply:
If the key does not exist at all I am almost certain that it still has it enabled. However I will push a change that will notify the administrator if the key does not exist so they can investigate further.

Reply:
Hey TBernard, we got to review the current status of the pull request today and it seems like we're just waiting for this change from you. Let me know if you need any help and I can take a look too!

Reply:
Hello Again,

Pushed changes to check for non-existent keys in my most recent commit.